Find gtk2 using pkg-config not gtk-config. gtk-config no longer exists.
--- configure.orig	2001-10-16 22:16:48.000000000 -0500
+++ configure	2022-03-19 23:04:14.000000000 -0500
@@ -2506,46 +2506,13 @@
 
 
 if test "$enable_gtk" != "no"; then
-	# Extract the first word of "gtk-config", so it can be a program name with args.
-set dummy gtk-config; ac_word=$2
-echo $ac_n "checking for $ac_word""... $ac_c" 1>&6
-echo "configure:2512: checking for $ac_word" >&5
-if eval "test \"`echo '$''{'ac_cv_prog_GTKCONFIG'+set}'`\" = set"; then
-  echo $ac_n "(cached) $ac_c" 1>&6
-else
-  if test -n "$GTKCONFIG"; then
-  ac_cv_prog_GTKCONFIG="$GTKCONFIG" # Let the user override the test.
-else
-  IFS="${IFS= 	}"; ac_save_ifs="$IFS"; IFS=":"
-  ac_dummy="$PATH"
-  for ac_dir in $ac_dummy; do
-    test -z "$ac_dir" && ac_dir=.
-    if test -f $ac_dir/$ac_word; then
-      ac_cv_prog_GTKCONFIG="yes"
-      break
-    fi
-  done
-  IFS="$ac_save_ifs"
-  test -z "$ac_cv_prog_GTKCONFIG" && ac_cv_prog_GTKCONFIG="no"
-fi
-fi
-GTKCONFIG="$ac_cv_prog_GTKCONFIG"
-if test -n "$GTKCONFIG"; then
-  echo "$ac_t""$GTKCONFIG" 1>&6
-else
-  echo "$ac_t""no" 1>&6
-fi
-
-
-	if test "X$GTKCONFIG" = X"yes"; then
 		cat >> confdefs.h <<\EOF
 #define USE_GTK 1
 EOF
 
-		GTK_CFLAGS="`gtk-config --cflags`"
-		WIDGET_LIBS="$WIDGET_LIBS `gtk-config --libs`"
+		GTK_CFLAGS="`pkg-config gtk+-2.0 --cflags`"
+		WIDGET_LIBS="$WIDGET_LIBS `pkg-config gtk+-2.0 --libs`"
 		WIDGET_OBJS="$WIDGET_OBJS gtk.o"
-	fi
 fi
 
 
